config/locales/es.yml

```diff
@@ -750,20 +750,21 @@ es:
     login_failed: No se ha podido iniciar la sesión, error de autenticación.
     login_name: Validación
     logout: Cerrar sesión
-    logs:
+    logs: "Logs"
     look_for_similar_items: Buscar artículos similares
     make_refund: Realizar devolución
-    make_sure_the_above_reimbursement_amount_is_correct:
-    manage_promotion_categories:
-    manage_variants:
-    manual_intervention_required:
+    make_sure_the_above_reimbursement_amount_is_correct: Asegúrese de que la
+    cantidad a devolver es la correcta
```

Comment:
Oops! I didn't do this on purpose. I guess it was Vim… That shouldn't be a new line, really.